tensorflow/core/kernels/unique_op.cc

```diff
@@ -76,7 +76,13 @@ struct UniqueOpHashMap<bfloat16, TIndex> {
 template <typename T, typename TIndex>
 class UniqueOp : public OpKernel {
  public:
-  explicit UniqueOp(OpKernelConstruction* context) : OpKernel(context) {}
+  static constexpr size_t INIT_SIZE = 1 << 12;
```

Comment:
This should follow Google naming conventions (https://google.github.io/styleguide/cppguide.html)